### PR TITLE
Add DefiLlama to DeFi page

### PR DESCRIPTION
**Pull request by 0xngmi**
change is trivial

### CODE REVIEW

File: src/content/defi/index.md [hunk 1]
```diff
@@ -374,6 +374,7 @@ todo -->
 
 ### DeFi data {#defi-data}
 
+- [DeFi Llama](https://defillama.com/)
 - [DeFi Pulse](https://defipulse.com/)
 - [DeFi Prime](https://defiprime.com/)
```
Comment (minimalsm) [inline src/content/defi/index.md:379]:
```suggestion
- [DeFi Pulse](https://defipulse.com/)
- [DeFi Prime](https://defiprime.com/)
- [DeFi Llama](https://defillama.com/)
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Hey @0xngmi thanks for the PR. Just making a small tweak as per [our listing policy](https://ethereum.org/en/contributing/adding-products/#product-ordering) 

> Unless products are specifically ordered otherwise, such as alphabetically, products will be displayed from most to least recently added to the page. In other words, the newest products get added to the bottom of the list.

I'll commit this change and merge 🚀

**Comment by minimalsm** [comment]
Merging 🚀 Remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @0xngmi for docs

**Comment by 0xngmi** [comment]
> Just making a small tweak as per our listing policy

Sorry about not following this, wasn't aware

**Comment by minimalsm** [comment]
@0xngmi no stress! Just wanted to let you know. Keep up the great working build out DefiLlama 😀

**Comment by 0xngmi** [comment]
download

**Pull request merged**